lib/tower_error_tracker/reporter.ex

```diff
@@ -19,6 +27,13 @@ defmodule TowerErrorTracker.Reporter do
     :ok
   end
 
+  def report_event(%Tower.Event{kind: :message, reason: reason, level: level} = event) do
+    TowerErrorTracker.ReportedMessage.exception(level: level, message: reason)
+    |> ErrorTracker.report([], context(event))
```

Comment:
Just realized that one drawback from this is that ErrorTracker would group every ReportedMessage together, as occurrences of the same error because it groups by "kind, source_line and source_function", and in this case we would be sending:
kind = TowerErrorTracker.ReportedMessage
source_line = ""
source_function =""